CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## 2.18.0-rc.0 / 2020-04-24
+
+* [CHANGE] Federation: It does not use remote storages anymore as designed initially. #7077
```

Comment:
Can we use a positive sentence here?

```suggestion
* [CHANGE] Federation: Only use local TSDB for federation (ignore remote read). #7077
```

Reply:
"Federation now only uses local TSDB" would be clearer.